book/src/filters.md

```diff
@@ -20,25 +20,28 @@ Enable it with Cargo features (see below for more information).
 **Table of contents**
 
 * **[Built-in filters][#built-in-filters]:**  
-  [`abs`][#abs],
-  [`as_ref`][#as_ref],
-  [`capitalize`][#capitalize],
-  [`center`][#center],
-  [`deref`][#deref],
-  [`escape|e`][#escape],
-  [`filesizeformat`][#filesizeformat],
-  [`fmt`][#fmt],
-  [`format`][#format],
-  [`indent`][#indent],
-  [`join`][#join],
-  [`linebreaks`][#linebreaks],
-  [`linebreaksbr`][#linebreaksbr],
-  [`lower|lowercase`][#lower],
-  [`safe`][#safe],
-  [`trim`][#trim],
-  [`truncate`][#truncate],
-  [`upper|uppercase`][#upper],
-  [`wordcount`][#wordcount]
+
+  * [`abs`][#abs],
+  * [`as_ref`][#as_ref],
+  * [`capitalize`][#capitalize],
+  * [`center`][#center],
+  * [`deref`][#deref],
+  * [`escape|e`][#escape],
+  * [`filesizeformat`][#filesizeformat],
+  * [`fmt`][#fmt],
+  * [`format`][#format],
+  * [`indent`][#indent],
+  * [`join`][#join],
+  * [`linebreaks`][#linebreaks],
+  * [`linebreaksbr`][#linebreaksbr],
+  * [`lower|lowercase`][#lower],
+  * [`safe`][#safe],
+  * [`string`][#string],
+  * [`title`][#title],
+  * [`trim`][#trim],
+  * [`truncate`][#truncate],
+  * [`upper|uppercase`][#upper],
+  * [`wordcount`][#wordcount]
```

Comment:
Would prefer the other additions to this be in a separate commit.

Reply:
Done!